components/about/KofiWidget.vue

```diff
@@ -0,0 +1,53 @@
+<template>  
+  <div class="kofi-widget">
+    <v-container>
+      <v-row>
+        <v-col sm="12" class="kofi-widget-content-wrap">
+          <div class="kofi-widget-content">
+            <v-row>
+              <h2>Buy us coffee ☕</h2>
+            </v-row>
+            <v-row>
+              <p>We are running this database after our 9-5 jobs. Help us stay awake!</p>
```

Comment:
you said you're moving the translations in a separate PR correct?

Reply:
Yeah, my plan was to merge this, then spin out the string ids after. My logic there is - we're unlikely to have translations for any of this off the bat anyways, until someone translates it, so a fast-follow should be fine, like we're doing with News.

Reply:
Ah fast-follow is good 👍